packages/rancher-monitoring/generated-changes/patch/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml.patch

```diff
@@ -0,0 +1,54 @@
+--- charts-original/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml
++++ charts/charts/hardenedKubelet/templates/pushprox-servicemonitor.yaml
+@@ -13,6 +13,51 @@
+     params:
+       _scheme: [https]
+     {{- end }}
++    honorLabels: true
++{{- if .Values.metricRelabelings }}
++    metricRelabelings:
++{{ tpl (toYaml .Values.metricRelabelings | indent 4) . }}
++{{- end }}
++{{- if .Values.relabelings }}
++    relabelings:
++{{ toYaml .Values.relabelings | indent 4 }}
++{{- end }}
++{{- if .Values.cAdvisor }}
++  - port: metrics
++    proxyUrl: {{ template "pushProxy.proxyUrl" . }}
++    {{- if .Values.clients.https.enabled }}
++    params:
++      _scheme: [https]
++    {{- end }}
++    path: /metrics/cadvisor
++    honorLabels: true
++{{- if .Values.metricRelabelings }}
++    metricRelabelings:
++{{ tpl (toYaml .Values.metricRelabelings | indent 4) . }}
++{{- end }}
++{{- if .Values.relabelings }}
++    relabelings:
++{{ toYaml .Values.relabelings | indent 4 }}
++{{- end }}
++{{- end }}
++{{- if .Values.probes }}
++  - port: metrics
++    proxyUrl: {{ template "pushProxy.proxyUrl" . }}
++    {{- if .Values.clients.https.enabled }}
++    params:
++      _scheme: [https]
++    {{- end }}
++    path: /metrics/probes
++    honorLabels: true
++{{- if .Values.metricRelabelings }}
++    metricRelabelings:
++{{ tpl (toYaml .Values.metricRelabelings | indent 4) . }}
++{{- end }}
++{{- if .Values.relabelings }}
++    relabelings:
++{{ toYaml .Values.relabelings | indent 4 }}
++{{- end }}
++{{- end }}
```

Comment:
Instead of patching pushprox specifically for hardenedKubelet, could we just update the rancher-pushprox chart to have `.Values.serviceMonitor.metricsConfig` and insert `{{ tpl (toYaml .Values.serviceMonitor.additionalEndpoints) | indent 2 }}` at the bottom of:

https://github.com/rancher/charts/blob/7836aff0c1b2846c3fbcf8ed2adc6963d21c20b1/charts/rancher-pushprox/rancher-pushprox/0.1.300/templates/pushprox-servicemonitor.yaml#L10-L15

We can then add this configuration to the values.yaml. This should come after `{{ tpl (toYaml .Values.serviceMonitor.additionalMetricsEndpointConfig) | indent 4 }}` from above.